src/generate/handlebarsAmfHelpers.test.ts

```diff
@@ -142,10 +150,14 @@ describe("HandlebarsAmfHelpers", () => {
     });
 
     it("returns an array of required and optional parameters on model with required, optional and additional parameters", () => {
-      const property1: model.domain.PropertyShape = new model.domain.PropertyShape();
-      const property2: model.domain.PropertyShape = new model.domain.PropertyShape();
-      const property3: model.domain.PropertyShape = new model.domain.PropertyShape();
-      const property4: model.domain.PropertyShape = new model.domain.PropertyShape();
+      const property1: model.domain.PropertyShape =
```

Comment:
why are there so many formatting changes?

Reply:
I updated the dependencies, apparently that included changes to the linting rules.